integration-tests/cypress/cypress.spec.js

```diff
@@ -55,7 +54,7 @@ moduleType.forEach(({
   testCommand
 }) => {
   // cypress only supports esm on versions >= 10.0.0
-  if (type === 'esm' && semver.satisfies(version, '<10.0.0')) {
+  if (type === 'esm' && version === '6.7.0') {
```

Comment:
unrelated fix: `version` will either be `6.7.0` or `latest`. `semver` should not work with `latest`